scripts/measures/2022/import-quality-measures.js

```diff
@@ -268,7 +268,7 @@ function addMultiPerformanceRateStrata(measures, strataRows) {
   _.each(strataRows, row => {
     const measureId = mapInput(row.measureId, 'measureId');
     const stratumName = row.stratumName.trim();
-    const description = row.longDescription.trim();
+    const description = row.longDescription.trim().replace(/\r/g, '');
```

Comment:
Carriage return character (`\r`) are appearing for strata 370 now. Since these were not in the original data, they need to be removed.